assets/src/edit-story/components/form/media.js

```diff
@@ -132,98 +132,104 @@ const LoadingDots = styled.div`
+  ) => {
+    const isMultiple = value === MULTIPLE_VALUE;
+    const openMediaPicker = useMediaPicker({
+      title,
+      buttonInsertText,
+      onSelect: onChange,
+      type,
+    });
 
-  const dropdownOptions = [
-    { name: __('Edit', 'web-stories'), value: 'edit' },
-    { name: __('Reset', 'web-stories'), value: 'reset' },
-  ];
+    const dropdownOptions = [
+      { name: __('Edit', 'web-stories'), value: 'edit' },
+      { name: __('Reset', 'web-stories'), value: 'reset' },
+    ];
 
-  const onOption = useCallback(
-    (opt, evt) => {
-      switch (opt) {
-        case 'edit':
-          openMediaPicker(evt);
-          break;
-        case 'reset':
-          onChange(null);
-          break;
-        default:
-          break;
-      }
-    },
-    [onChange, openMediaPicker]
-  );
+    const onOption = useCallback(
+      (opt, evt) => {
+        switch (opt) {
+          case 'edit':
+            openMediaPicker(evt);
+            break;
+          case 'reset':
+            onChange(null);
+            break;
+          default:
+            break;
+        }
+      },
+      [onChange, openMediaPicker]
+    );
 
-  const ref = useRef();
-  const [isHovering, setIsHovering] = useState(false);
-  const [isFocused, setIsFocused] = useState(false);
-  const resettableProps = {
-    tabIndex: 0,
-    'aria-label': ariaLabel,
-    onFocus: () => setIsFocused(true),
-    onBlur: (evt) => setIsFocused(ref.current.contains(evt.relatedTarget)),
-    onPointerEnter: () => setIsHovering(true),
-    onPointerLeave: () => setIsHovering(false),
-  };
+    const ref = useRef();
+    const [isHovering, setIsHovering] = useState(false);
+    const [isFocused, setIsFocused] = useState(false);
+    const resettableProps = {
+      tabIndex: 0,
+      'aria-label': ariaLabel,
+      onFocus: () => setIsFocused(true),
+      onBlur: (evt) => setIsFocused(ref.current.contains(evt.relatedTarget)),
+      onPointerEnter: () => setIsHovering(true),
+      onPointerLeave: () => setIsHovering(false),
+    };
 
-  const isMenuVisible = isHovering || isFocused;
+    const isMenuVisible = isHovering || isFocused;
 
-  return (
-    <Container
-      ref={ref}
-      className={`${className}`}
-      disabled={disabled}
-      circle={circle}
-      size={size}
-      {...rest}
-      {...(canReset && resettableProps)}
-    >
-      {value && !isMultiple ? (
-        <Img src={value} circle={circle} alt={alt} />
-      ) : (
-        <DefaultImage size={size} />
-      )}
-      {loading && <LoadingDots />}
-      {canReset && isMenuVisible && (
-        <DropDownMenu options={dropdownOptions} onOption={onOption} />
-      )}
-      {!canReset && (
-        <EditBtn
-          onClick={openMediaPicker}
-          circle={circle}
-          aria-label={ariaLabel}
-        >
-          <EditIcon />
-        </EditBtn>
-      )}
-    </Container>
-  );
-}
+    return (
+      <Container
+        ref={ref}
+        className={`${className}`}
```

Comment:
Thanks for changing all of the others 🙏 
```suggestion
        className={className}
```